fastlane/lib/fastlane/actions/xcodebuild.rb

```diff
@@ -636,6 +648,47 @@ def self.author
         "dtrenz"
       end
     end
+
+    class XccreatexcframeworkAction < Action
+      def self.run(params)
+        params_hash = { create_xcframework: true }.merge(params)
+
+        XcodebuildAction.run(params_hash)
+      end
+
+      def self.example_code
+        [
+          'xccreatexcframework(
+            framework: ["FrameworkA.framework", "FrameworkB.framework"],
```

Comment:
Typo:
```suggestion
            frameworks: ["FrameworkA.framework", "FrameworkB.framework"],
```

Reply:
Gonna try to create an independent action. As the `-create-xcframework` is quite independent from the rest of the `xcodebuild` commands, and most (if not all) of the parameters are exclusive to this action, I think it makes sense to do it. In addition having an independent action will let me easily include the parameters I had to leave unsupported because it would have made the logic to integrate them with the existing parameters too complicated. Let me work on it for a day or two...